packages/components/app/styles/components/tabs.scss

```diff
@@ -38,26 +38,18 @@ $indicator-height: 3px;
   overflow-x: auto;
   white-space: nowrap;
   list-style: none;
+  background: linear-gradient(to right, var(--token-color-border-primary), var(--token-color-border-primary)) repeat-x;
+  background-position: bottom floor(calc($indicator-height / 2)) right 0;
+  background-size: 20px 1px;
```

Comment:
OK, found it: it's in this PR
https://github.com/hashicorp/design-system/pull/581/commits/8d6be094153ff899ad8aacfad97a2c1ca878e059
which is not related to the overflow and was merged without review.

I know it's a big ask, but what do you think of removing this code related to the overflow, and then add it in another PR so it can be properly reviewed?

Reply:
ok I'll delete it and see

Reply:
I deleted everything related to overflow for now. Will look at it again and push up later as new commit or maybe separate PR